pkgs/by-name/an/anubis/package.nix

```diff
@@ -58,12 +76,12 @@ buildGoModule (finalAttrs: {
   ]
   ++ lib.optionals stdenv.hostPlatform.isLinux [ "-extldflags=-static" ];
 
-  postPatch = ''
-    patchShebangs ./web/build.sh
-  '';
-
   preBuild = ''
-    go generate ./... && ./web/build.sh && cp -r ${finalAttrs.xess}/xess.min.css ./xess
+    go generate ./...
```

Comment:
We should run this after copying the assets into place just to be sure, so that we consult mimic upstream.